### PR TITLE
python310Packages.readability-lxml: disable broken test on darwin

### PR DESCRIPTION
**Pull request by keegancsmith**
## Description of changes

This test is broken on darwin. The test is fixed in master for readability-lxml, but the fix requires pulling in a few dependencies which are not packaged in nixpkgs.

My goal is to have no dependencies of qutebrowser marked as broken.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/readability-lxml/default.nix [hunk 1]
```diff
@@ -31,8 +31,12 @@ buildPythonPackage rec {
     timeout-decorator
   ];
 
+  disabledTests = [
+    # Test is broken on darwin
```
Comment (natsukium) [inline pkgs/development/python-modules/readability-lxml/default.nix:35]: Could you please include the URL of the fixed PR in this comment?

Comment (keegancsmith) [reply pkgs/development/python-modules/readability-lxml/default.nix]: done

**Comment by natsukium** [comment]
The change looks good to me.
Would you please rewrite the commit message as `python310Packages.readability-lxml: disable broken test on darwin`?

**Comment by keegancsmith** [comment]
@natsukium done

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 250407` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python310Packages.readability-lxml</li>
    <li>python310Packages.readability-lxml.dist</li>
    <li>python311Packages.readability-lxml</li>
    <li>python311Packages.readability-lxml.dist</li>
    <li>qutebrowser</li>
    <li>qutebrowser.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/readability-lxml/default.nix [hunk 2]
```diff
@@ -31,8 +31,12 @@ buildPythonPackage rec {
     timeout-decorator
   ];
 
+  disabledTests = [
```
Comment (natsukium) [inline pkgs/development/python-modules/readability-lxml/default.nix:34]:
```suggestion
  disabledTests = lib.optionals stdenv.isDarwin [
```
Sorry to bother you again, but this is only needed for darwin. Could you please add a conditional branch?

**Comment by keegancsmith** [comment]
Just committed your suggestion and manually tested on x86_64-linux and aarch64-darwin

**Review by natsukium — APPROVED** [review]
LGTM, thanks!

**Pull request merged**